code/datums/ruins.dm

```diff
@@ -0,0 +1,22 @@
+/datum/map_template/ruin
+	//name = "A Chest of Doubloons"
+	name = null
```

Comment:
Files can be renamed =P
Don't feel the need to do this change, it's too minor a suggestion to really matter.